api-extractor.json

```diff
@@ -88,9 +88,22 @@
      *
      * DEFAULT VALUE: no overrideTsconfig section
      */
-    // "overrideTsconfig": {
-    //   . . .
-    // }
+    "overrideTsconfig": {
```

Comment:
Can you add a comment here describing what is actually being overridden? (I assume it's the "include", but it's not obvious, and will become less so if tsconfig changes.)

Alternately, if this overridden tsconfig can be slimmed down, that could help.